src/react/Routes.jsx

```diff
@@ -42,6 +43,10 @@ function Routes() {
                             selected: !!matchPath(pathname, { path: '/buckets' }) ||
                                       !!matchPath(pathname, { path: '/create-bucket' }),
                         },
+                        {
+                            link: <Link to="/workflows">Workflows</Link>,
```

Comment:
According to the design:
```suggestion
                            link: <Link to="/workflows">Workflow</Link>,
```